### DISCUSSION TITLE
화면공유 시 전체화면 / 화면 전환 이슈

### DISCUSSION THREAD

**Discussion by joorijung** [metamong-friends/metamong-friends.github.io#37] [제안하기]
1) 화면공유할 때 화면 전체화면이 안되는데 전체로 보고싶습니다! 너무 화면이 작아요 ㅠㅠ
2) 화면 공유 되고 있을 때 grid view/map view를 누르면 갑자기 화면 공유 view 버튼이 없어지네요! 남아있으면 좋겠어요.

**Comment by ACT-TJ** [comment]
안녕하세요!
두 의견 모두 viewer ( 화면공유를 받는 입장) 에서의 의견으로 이해를 하고 답변을 드릴게요. 혹시 제가 잘못 이해한 부분이 있다면 또 커맨트 남겨주세요 ^^

1. 네 개선의견으로 좀 더 크게 가져갈 수 있는 부분 검토해 보겠습니다^^ . 하단에 비디오 영역을 노출하고, 채팅, 이모지를 띄우는 영역들을 고려해서 전체로 가져가면 공유하는 컨텐츠가 너무 잘리는 경우들이 있있습니다. 해당 상황들을 고려하되 의견 참고해 개선 고민해 보도록 하겠습니다^^

2. 우측상단의 세가지 중 좌측 아이콘 말씀이신 것 같습니다. 공유를 받는 중 다른 뷰로 이동했을 때, 할 수 있는 옵션이
많지만 크게 아래 케이스 들이 있을 것 같은데요.

. 전환한 뷰 (그리드나 맵뷰) 사용
. 다시 해당 스크린 쉐어 뷰로 이동
. 다른 유저의 스크린 쉐어 뷰로 이동
. 내가 스크린 쉐어를 시작

갈 수 있는 많은 케이스들이 있고 맵뷰나 그리드뷰로 이동했는데 계속 특정 다른 유저가 스트리밍 하고있다는 표시를
해당 아이콘으로 하는 것은 혼선이 있는 케이스들이 있어 제거를 하고, 다시 해당유저의 스크린쉐어 아이콘을
클릭하면 재수신이 되도록 흐름을 잡아 두었습니다.

이 부분은 좀 더 저희도 사용성 검증과 유저분들의 의견을 수렴해 본 뒤 검토해 볼 수 있도록 하겠습니다^^

좋은 의견 너무 감사드립니다!!

**Comment by shlee303** [comment]
@joorijung 1에 대해 전체화면 기능 반영하였습니다. 2번은 여러명이 화면공유를 할 수 있는 user flow 상 제외된 부분이어서 양해 부탁드립니다! 다른 방법을 좀 더 생각해보겠습니다 .항상 좋은 의견 감사합니다~!!!
